.github/workflows/build.yml

```
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - main
-      - release/**
+      - release/[0-9].[0-9]+.x*
```

I think this is ok under the rules for GitHub's weird [filter syntax](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), but we might want to quote it for robustness anyways?

```suggestion
      - 'release/[0-9].[0-9]+.x*'
```